build.sbt

```diff
@@ -55,6 +60,8 @@ val timeBenchmark =
     .enablePlugins(JmhPlugin)
     .settings(commonSettings: _*)
     .settings(
+      // FIXME: does not compile with dotty
+      crossScalaVersions := List(scalaVersion.value),
```

Comment:
I would classify it as a bug. The Java parser of Dotty should be fixed. I'll do a PR shortly. But how can we used a fixed Dotty in strawman nightly builds? /cc @smarter